javascript-and-json/main.js

```diff
@@ -0,0 +1,33 @@
+var book = [
+  {
+    isbn: '123456789',
+    title: 'Harry Potter',
+    author: 'JK Rowling'
+  },
+  {
+    isbn: '246802468',
+    title: 'Percy Jackson',
+    author: 'Rick Riordan'
+  },
+  {
+    isbn: '135790159',
+    title: 'Demon King',
+    author: 'Cinda Williams'
+  }
+];
+
+console.log('value of book: ', book);
+console.log('value of typeof book: ', typeof book);
+var jsonBook = JSON.stringify(book);
+console.log('value of jsonBook: ', jsonBook);
+console.log('value of typeof jsonBook: ', typeof jsonBook);
+var student = {
```

Comment:
you don't want to stringify this object, you want to manually create a JSON string.